src/frontend/helpers/gamepad.ts

```diff
@@ -411,6 +411,15 @@ export const initGamepad = () => {
 
     currentController = controllerIndex
     dispatchControllerEvent(gamepad.id)
+
+    window.addEventListener(
+      'mousemove',
+      () => {
+        currentController = -1
+        dispatchControllerEvent('')
+      },
+      { once: true }
```

Comment:
only listen for the first mouse movement after a controller is activated, we don't need to listen to that constantly